dev/tox.ini

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 
 [pycodestyle]
-ignore=E226,E241,E305,E402,E722,E731,E741,W503,W504
+ignore=E226,E241,E305,E402,E722,E731,E741,W503,W504,W605
 max-line-length=100
 exclude=cloudpickle.py,heapq3.py,shared.py,python/docs/conf.py,work/*/*.py,python/.eggs/*,dist/*
+[pydocstyle]
+ignore=D100,D101,D102,D103,D104,D105,D106,D107,D200,D201,D202,D203,D204,D205,D206,D207,D208,D209,D210,D211,D212,D213,D214,D215,D300,D301,D302,D400,D401,D402,D403,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```

Comment:
This is tiny, but is this blank line necessary? it seems to separate the heading from its content?

Reply:
Between [pydocstyle] to ignore is necessary and standard style. Post ignore the blank line was  added as per requested by last reviewer on previous PR - https://github.com/apache/spark/pull/20556#discussion_r168079061

Reply:
I don't think that's what @ueshin was asking for, I think it was a blank line after the `ignore=...`, but if @ueshin is around we can see what @ueshin says. It's also relatively minor provided everything functions.

Reply:
I just asked to add a line break at the end of file, and the current style looks good to me.

Reply:
Oh I was asking about the blank line at line 21. Isn't "ignore=" related to "[pydocstyle]"? compare to no blank line between 16-17. It's really trivial but hey.